user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.md

```diff
@@ -16,9 +16,17 @@ With this feature, users can:
 
 - Access files or webpages on the DMA that have been made available for remote access.<!-- RN 38426 -->
 
-  To access such files, use the remote access URL followed by `/public/` (e.g. the file *image.png* via `https://ziine-skyline.on.dataminer.services/public/image.png`).
+  The files or webpages that are available for public access are located in the following folders on the DMA:
```

Comment:
If those other folders are only intended for internal use within DataMiner, it might be better if we do not mention them here. What do you think?

Reply:
I'd indeed not mention them.

But maybe remove the "or webpages" from the sentence?
Cause otherwise customers might think that the dashboards and other apps also reside in those folders which is not true.
(from my side it can be left as is)

Reply:
OK, I'll change this to prevent confusion.